CMakeLists.txt

```diff
@@ -299,7 +299,7 @@ endif()
 include_directories(SYSTEM ${FOLLY_INCLUDE_DIRS})
 
 if(NOT ${VELOX_BUILD_MINIMAL})
-  find_package(Protobuf 3.0.0 REQUIRED)
+  find_package(Protobuf 3.21.0 REQUIRED)
```

Comment:
We need to ensure that 3.21.0 works well centos 7  , else this is a no go. Can we validate that 3.21 is available for centos 7 (ideally without having to build from source).

Reply:
> We need to ensure that 3.21.0 works well centos 7 , else this is a no go. Can we validate that 3.21 is available for centos 7 (ideally without having to build from source).

I changed "3.21.0" -> "3.0.0" now.